recipes/qikpropservice/meta.yaml

```diff
@@ -0,0 +1,60 @@
+# Note: there are many handy hints in comments in this example -- remove them when you've finalized your recipe
+
+# Jinja variables help maintain the recipe as you'll update the version only here.
+# Using the name variable with the URL in line 14 is convenient
+# when copying and pasting from another recipe, but not really needed.
+{% set name = "qikpropservice" %}
+{% set version = "0.2.3" %}
+{% set sha256 = "fb911ede6b8cdafb8350db841299aec24ea40947832bb54ca8665ae1ccd3c6d8" %}
```

Comment:
Please more the checksum to the key below. BTW, try grayskull to create recipes. It will create one that is closer to a format that is ready to be merged.